binding/web/scripts/setup_test.js

```diff
@@ -51,10 +42,6 @@ try {
     fs.copyFileSync(join(paramsSourceDirectory, file), join(outputDirectory, file));
   });
 
-  fs.readdirSync(paramsSourceDirectory).forEach(file => {
-    fs.copyFileSync(join(paramsSourceDirectory, file), join(outputDirectory, file));
-  });
-
   fs.copyFileSync(testDataSource, join(outputDirectory, 'test_data.json'));
 
   fs.mkdirSync(join(outputDirectory, 'keywords'), { recursive: true });
```

Comment:
I think the idea is to dumb copy all the keyword files, since when additional tests are added to `test_data.json` we don't want to have to go in and update all these copy scripts